setup.py

```diff
@@ -28,6 +28,6 @@ def limit_major_minor_version(version: str) -> str:
     package_dir={'': 'src'},
     packages=find_packages(where='src'),
     install_requires=[
-        f"profiles-rudderstack=={limit_major_minor_version(version)}",
+        f"profiles-rudderstack==0.11.3", 'pandas>=1.4.3', 'matplotlib>=3.7.1', 'cexprtk==0.4.1'
```

Comment:
```suggestion
        f"profiles-rudderstack=={limit_major_minor_version(version)}", 'pandas>=1.4.3', 'matplotlib>=3.7.1', 'cexprtk==0.4.1'
```

The `version` is set to 0.12.0, so we don't need to change this.